<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeDetecDialog</class>
 <widget class="QDialog" name="changeDetecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="CardWidget" name="CardWidget_4">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="BodyLabel" name="BodyLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Select Pre Raster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="preTifComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_9">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="BodyLabel" name="BodyLabel_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Select Late Raster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="lateTifComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="CheckBox" name="onlyInferExtentCb">
       <property name="text">
        <string>Infer Only Area of Interest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="extentLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="drawExtentPb">
       <property name="text">
        <string>Draw on the Map</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="BodyLabel" name="BodyLabel_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Image Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="imgSizeRb_512">
        <property name="text">
         <string>512</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">imgSizeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="imgSizeRb_1024">
        <property name="text">
         <string>1024</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">imgSizeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="imgSizeRb_2048">
        <property name="text">
         <string>2048</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">imgSizeButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="BodyLabel" name="BodyLabel_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Overlap(%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SpinBox" name="overlapSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_5">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="BodyLabel" name="BodyLabel_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Simply Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="simplyRb_low">
        <property name="text">
         <string>Low</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">simplyButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="simplyRb_mid">
        <property name="text">
         <string>Medium</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">simplyButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="RadioButton" name="simplyRb_high">
        <property name="text">
         <string>High</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">simplyButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_6">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="BodyLabel" name="BodyLabel_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SpinBox" name="removeAreaSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_7">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="BodyLabel" name="BodyLabel_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Hole</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SpinBox" name="removeHoleSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_8">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="BodyLabel" name="BodyLabel_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Select Graphics Card</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="selectGpuComboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BodyLabel" name="BodyLabel_4">
        <property name="text">
         <string>Select Results Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="saveLE"/>
      </item>
      <item>
       <widget class="ToolButton" name="selectSaveFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="PushButton" name="runPB">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="imgSizeButtonGroup"/>
  <buttongroup name="simplyButtonGroup"/>
 </buttongroups>
</ui>
